module/zfs/metaslab.c

```diff
@@ -4640,7 +4640,7 @@ find_valid_metaslab(metaslab_group_t *mg, uint64_t activation_weight,
 	if (msp == NULL)
 		msp = avl_nearest(t, idx, AVL_AFTER);
 
-	int tries = 0;
+	uint32_t tries = 0;
```

Comment:
Why uint32_t if zfs_metaslab_find_max_tries is uint_t?

Reply:
It is an artifact of my treatment of `uint32_t` as a short hand for `unsigned int`. I will change it.